src/snakemake/ioutils/__init__.py

```diff
@@ -25,5 +26,6 @@ def register_in_globals(_globals):
             "parse_input": parse_input,
             "extract_checksum": extract_checksum,
             "flatten": flatten,
+            "choose_tmp": choose_tmp,
```

Comment:
I think in addition this needs to be present in the eval environment for resource expressions, right?